### ISSUE TITLE
[a11y] The page doesn't have any frame heading to share the context/overview for screenreaders

### ISSUE THREAD

**Issue by plabanTest1** [civiform/civiform#9615]
**Describe the bug**
The page doesn't have a level-one heading

**To Reproduce**

1. Go to https://qa.civiform.dev/programs
2. Click on DevTools 
3. Select Log in as: Civiform Admin
4. Click on the View button on any of the listed programs.
5. Run the Voiceover to the element and verify the heading.

**Expected behavior**
Ensure that the page, or at least one of its frames contains a level-one heading as per GAR guidelines.

**Screenshots**

![Image](https://github.com/user-attachments/assets/b7408b8d-3fda-4d09-88ed-08d135bdceaf)

**Desktop (please complete the following information):**

- OS: [iOS]
- Browser [Chrome]
- Version [e.g. 132.0.6834.111]

**Additional context**

**Comment by shreyachatterjee00** [comment]
@gwendolyngoetz Confirming that we don't have any h1s on the screen page?

**Comment by gwendolyngoetz** [comment]
No, there never has been one on the view screen. I think there are two on the edit screen. But we know the admin side is not accessible.
